app/static/src/app/components/options/ParameterSetView.vue

```diff
@@ -15,6 +15,13 @@
                      type="eye"
                      @click="toggleHidden"
                      v-tooltip="'Show Parameter Set'"></vue-feather>
+        <vue-feather class="inline-icon clickable swap-param-set ms-2"
+                     type="shuffle"
+                     :disabled="!canSwapParameterSet"
+                     :stroke="canSwapParameterSet ? 'black' : 'lightgray'"
+                     :style="{ cursor: canSwapParameterSet ? 'pointer' : 'default' }"
+                     @click="swapParameterSet"
+                     v-tooltip="'Swap Parameter Set with Current Parameter Values'"></vue-feather>
```

Comment:
Might be worth changing this to say "Can't swap with current values as run is out of date" when disabled?

Reply:
i am just working on an upgrade for tooltips currently for the min/max validation ticket, if possible lets hold on to this change, maybe i can make a small ticket because I will need to refactor how tooltips are written anyway and I can add this in as part of that branch!

Reply:
sounds sensible - please make a ticket so we can keep track of it